### PR TITLE
[#251] multilabel classifier

### PR DESCRIPTION
**Pull request by Samuel-Campbell**
[#251]

1) Added multi class classifier
2) Added unittest
3) Revised every single ML unittest and fixed some of them
4) ready to train

### CODE REVIEW

File: src/ml_service/model_training/abstract_models/abstract_classifier.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import numpy as np
+from util.log import Log
+from sklearn.metrics import precision_recall_fscore_support
+
+
+class AbstractClassifier:
```
Comment (arekmano) [inline src/ml_service/model_training/abstract_models/abstract_classifier.py]: IMO this is unnecessary Abstraction. While the classifiers may have the same methods, their implementations can vary. I.e. get_weights is assuming that Scikit will be used. Same with test. These effectively block off Neural networks from being used (Tensorflow, Keras, etc)

**Review by arekmano — DISMISSED** [review]

File: src/ml_service/model_training/abstract_models/abstract_classifier.py (hunk 1 above)
Comment (Samuel-Campbell) [reply src/ml_service/model_training/abstract_models/abstract_classifier.py]: You can just overwrite methods. I essentially wanted us to use a template

**Review by dodoels — APPROVED** [review]
Look clean to me, can't find any issue.

File: src/ml_service/model_training/abstract_models/abstract_classifier.py (hunk 1 above)
Comment (arekmano) [reply src/ml_service/model_training/abstract_models/abstract_classifier.py]: Still. Inheritance does not make much sense in the Python language. I will dismiss my review, but still stand by my original argument

**Pull request merged**